Python/Product/Django/TemplateParsing/DjangoFormatDefinitions.cs

```diff
@@ -24,8 +24,8 @@
 namespace Microsoft.PythonTools.Django.TemplateParsing {
     [Export(typeof(EditorFormatDefinition))]
     [ClassificationType(ClassificationTypeNames = DjangoPredefinedClassificationTypeNames.TemplateTag)]
-    [Name(DjangoPredefinedClassificationTypeNames.TemplateTag)]
-    [DisplayName(DjangoPredefinedClassificationTypeNames.TemplateTag)]
+    [Name(DjangoPredefinedClassificationTypeNames.TemplateTag)] // TODO: Localization - this string appears in fonts page in tools options
+    [DisplayName(DjangoPredefinedClassificationTypeNames.TemplateTag)] // TODO: Localization - not sure if this is used, does not affect fonts page
```

Comment:
I'm setting up a machine to see if the strings in fonts page get translated or not. Will know later today.

Looking at our code index, it doesn't look like DisplayName is being used by others that deriving from ClassificiationFormatDefinition.  If I make Name and DisplayName different, I can see that it's Name that is shown in fonts page.